database-migrations.md

```diff
@@ -107,4 +107,13 @@ Sometimes, migrations don't go quite as planned, and it'd be nice to roll them b
 php apex migrations:down --number NUMBER_TO_ROLL_BACK
 ```
 
-This will call all the `down()` methods in your executed migrations.  If you'd like to roll back all your migrations, simply don't pass the `--number` option.  If you'd like to roll back migrations outside of the console, you can call `IMigrator::rollBackMigrations()`.
\ No newline at end of file
+This will call all the `down()` methods in your executed migrations.  If you'd like to roll back all your migrations, simply don't pass the `--number` option.  If you'd like to roll back migrations outside of the console, you can call `IMigrator::rollBackMigrations()`.
+
+<h2 id="using-mysql">Using MySQL</h2>
+
+The migrator will use the PostgreSQL database adapter by default. If you need MySQL configured, you need to define an environment variable called `DB_DRIVER` to point to your database adapter.
+
+```bash

```

Comment:
Rather than showing a bash example, can you change this to be more in line with what's in the `.env.app.php` file?  Something like:

```php
Environment::setVar('DB_DRIVER', \Opulence\Databases\Adapters\Pdo\MySql\Driver::class);
```